README.md

```diff
@@ -382,6 +392,109 @@ ustream.channel.setBrandingType(channelId, type)
 | channelId         | int                | ID of an existing channel.    |
 | type              | string             | The branding type.            |
 
+## Playlist API
+
+### Is Enabled
+You can check whether playlists are enabled or disabled on the channel page.
+
+```JavaScript
+ustream.isEnable(channelId)
+```
+
+| Parameter         | Type               | Description                |
+|-------------------|--------------------|----------------------------|
+| channelId         | int                | ID of an existing channel. |
+
+> Promise returns a boolean result stating if playlists are enabled for the channel or not.
+
+
+### List Playlists
+Retrieve a list of the playlists in the channel
+
+```JavaScript
+ustream.playlist.list(channelId, pageSize, page, includeEmptyLists)
+```
+
+| Parameter         | Type               | Description                |
+|-------------------|--------------------|----------------------------|
+| channelId         | int                | ID of an existing channel. |
+| pageSize  | int             | How many entries to return in one request.  Default is `50` and max is `50`    |
+| page  | int             | Starting page number.  Default is `1`    |
+| includeEmptyLists | boolean | If the value is true then empty playlists will be returned (false by default) |
+
+> Promise returns a Pageable result. See "Paging Results" section for details.
+
+
+### List Videos in Playlist
+Retrieve a list of videos that are in a specific playlist
+
+```JavaScript
+ustream.playlist.listVideos(playlistId, pageSize, page)
+```
+
+| Parameter         | Type               | Description                |
+|-------------------|--------------------|----------------------------|
+| playlistId         | int                | ID of an existing playlist. |
+| pageSize  | int             | How many entries to return in one request.  Default is `200` and max is `200`    |
+| page  | int             | Starting page number.  Default is `1`    |
+
+
+> Promise returns a Pageable result. See "Paging Results" section for details.
+
+
+### Create Playlist
+Create a new playlist in the channel
+
+```JavaScript
+ustream.playlist.create(channelId, title, options)
+```
+
+| Parameter         | Type               | Description                |
+|-------------------|--------------------|----------------------------|
+| channelId         | int                | ID of an existing channel. |
+| title         | string                | The title of the playlist. |
+| is_enabled  | int             | Whether the playlist is enabled or not. Values: 1 (enabled), 0 (disabled). (1 by default)    |
```

Comment:
All variables should be in `camelCase`. `is_enabled` should be renamed to `isEnabled`.